source/trx2junit/Internal/JUnitBuilder.cs

```diff
@@ -58,6 +61,7 @@ private void AddTestSuite(string testSuiteName, IEnumerable<TestDefinition> test
         //---------------------------------------------------------------------
         private void AddTest(XElement xTestSuite, TestDefinition test)
         {
+            Debug.Assert(_lookup != null);
```

Comment:
Interestingly this works for nullability in Release too.
Otherwise there should be `_lookup![test.Id];` (which looks ugly :wink:).

But this needs tests in Debug config.

Reply:
Added with 221ddcacd7128ac43f7793f99d1552fcfea19ab4